rtengine/imagefloat.h

```diff
@@ -218,14 +218,14 @@ class Imagefloat : public IImagefloat, public ImageIO
         return (uint32_t) ((sign << 31) | (exponent << 23) | mantissa);
     }
 
-    virtual void         normalizeFloat(float srcMinVal, float srcMaxVal);
-    void                 normalizeFloatTo1();
-    void                 normalizeFloatTo65535();
-    void                 calcCroppedHistogram(const ProcParams &params, float scale, LUTu & hist);
-    void                 ExecCMSTransform2(cmsHTRANSFORM hTransform);
-
-    void                 ExecCMSTransform(cmsHTRANSFORM hTransform);
-    void                 ExecCMSTransform(cmsHTRANSFORM hTransform, const LabImage &labImage, int cx, int cy);
+    void normalizeFloat(float srcMinVal, float srcMaxVal) override;
+    void normalizeFloatTo1();
+    void normalizeFloatTo65535();
+    void calcCroppedHistogram(const ProcParams &params, float scale, LUTu & hist);
+    void ExecCMSTransform2(cmsHTRANSFORM hTransform);
+
+    void ExecCMSTransform(cmsHTRANSFORM hTransform);
+    void ExecCMSTransform(cmsHTRANSFORM hTransform, const LabImage &labImage, int cx, int cy);
```

Comment:
:+1: If you like, I can fire up clang-tidy to add `override`s everywhere.

Reply:
Please do. :+1:

Reply:
Processing...

Reply:
[Here](https://filebin.net/j87h1dkc7lb65ej2) it is. Take it with a grain of salt, though: `clang-tidy` has some built-in default checks you can't disable. So there's always a bit more done than expected. See the grouping of `NAN`s in `dcraw.cc` in the patch, where one grouping was missed.

HTH,
Flössie

Reply:
@Floessie Great. Shall I push?

Reply:
@heckflosse Ingo, sure. Maybe also add the braces around the first `NAN` block in `dcraw.cc` to make it symmetric.

Reply:
Ok, I'm just compiling it before I will push it...